### PR TITLE
data/images/db

### PR DESCRIPTION
**Pull request by paulms1**
set up of database + assets

### CODE REVIEW

**Review by ashleycoles — APPROVED** [review]
Looks good to me! Just make sure that before you merge this request you change it to merge into your story branch instead of master. Should be able to do that just under the pull request name

File: fe/index.php [hunk 1]
```diff
@@ -0,0 +1,132 @@
+
+<!DOCTYPE html>
+<html>
+    <head>
+        <link href="https://fonts.googleapis.com/css?family=Anton&display=swap" rel="stylesheet">
+        <link rel="stylesheet" type="text/css" href="normalize.css">
+        <link rel="stylesheet" type="text/css" href="styles.css">
```
Comment (robings) [inline fe/index.php:7]: add `<meta name="viewport" content="width=device-width">`

File: fe/styles.css [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+    padding-top: 60px;
+}
+
+header {
+    background-image: linear-gradient(
+            rgba(0, 0, 0, 0.2),
+            rgba(0, 0, 0, 0.2)
+    ), url('../images/recordsHero.jpg');
+    width: 80%;
+    height: 300px;
```
Comment (robings) [inline fe/styles.css:16]: could be better to have a minimum height rather than a height definition

File: fe/styles.css [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+    border: 5px solid RGB(255,255,255);
+}
+
+footer {
+    background-image: linear-gradient(
+            rgba(0, 0, 0, 0.2),
+            rgba(0, 0, 0, 0.2)
+    ), url('../images/recordsHero.jpg');
+    width: 80%;
+    height: 150px;
```
Comment (robings) [inline fe/styles.css:116]: as above, it might be better to have a min-height definition rather than height.

File: fe/styles.css [hunk 4]
```diff
@@ -0,0 +1,192 @@
...
+@media only screen and (max-width: 900px) {
+    .surround{
+        background: RGB(255,255,255);
+        background-size: 330px;
+    }
+}
+
+@media only screen and (max-width: 850px) {
+    img {
+        display: block;
```
Comment (robings) [inline fe/styles.css:130]: here the only thing that changes anything in the browser is the width definition.

File: data/data.txt [hunk 5]
```diff
@@ -0,0 +1,43 @@
+
```
Comment (robings) [inline data/data.txt:1]: is this a preparatory file? If so you could gitignore it

**Review by robings — CHANGES_REQUESTED** [review]
This looks good on all browsers. Just a few comments. Mainly it would be worth looking at the css, as some of it doesn't appear to affect the display.

**Pull request merged**